### PR TITLE
Playout delay

### PR DESCRIPTION
**Pull request by drkron**
I have been assigned to complete the work with the playout-delay PR. However, I didn't manage to add changes to that PR so instead I've create a new PR where the review comments have been addressed.

### CODE REVIEW

File: experiments/rtp-hdrext/playout-delay/index.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  ID   | len=2 |       MIN delay       |       MAX delay       |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+
+12 bits for Minimum and Maximum delay. This represents a range of 0 - 40950 milliseconds for minimum and maximum (with a granularity of 10 ms). A granularity of 10 ms is sufficient since we expect the following typical use cases:
+
+* 0: Certain gaming scenarios (likely without audio) where we will want to play the frame as soon as possible. Also, for remote desktop without audio where rendering a frame asap makes sense
+* 100/150/200: These could be the max target latency for interactive streaming use cases depending on the actual application (gaming, remoting with audio, interactive scenarios)
+* 400: Application that want to ensure a network glitch has very little chance of causing a freeze can start with a minimum delay target that is high enough to deal with network issues. Video streaming is one example.
```
Comment (alvestrand) [inline experiments/rtp-hdrext/playout-delay/index.md]: Are these numbers in milliseconds or in the 10-ms units used in the packet?

200 ms seems about right for max target gaming latency,  but 200 units (2 seconds) seems far too long.

Comment (drkron) [reply experiments/rtp-hdrext/playout-delay/index.md]: The unit is ms, will add that.

**Comment by jimmywarting** [comment]
Could you advice some help of how to configure the playout delay with javascript?

I would want to let the reciver to see the video 10s later if possible

**Comment by drkron** [comment]
> Could you advice some help of how to configure the playout delay with javascript?

I don't think it's exposed in any JavaScript API at the moment.

> I would want to let the reciver to see the video 10s later if possible

The intended use case is to remotely configure the receiver's jitter buffer. I don't think it would work to set it that high. Anything higher than 1-2 seconds will most likely not work.

**Comment by kuddai** [comment]
> Could you advice some help of how to configure the playout delay with javascript?
> 
> I would want to let the reciver to see the video 10s later if possible

Could you please describe what was your usecase? There is an ongoing (but it finishes soon) [origin-trial](https://developers.chrome.com/origintrials/#/view_trial/1463669878895411201) that allows to accomplish that in javascript.

**Comment by jimmywarting** [comment]
> @kuddai Could you please describe what was your usecase? There is an ongoing (but it finishes soon) origin-trial that allows to accomplish that in javascript. 

I'm a gymnast and when i step of the floor or mattress i want to go to a nearby device that shows my lap with a 10 to 20 seconds delay while the other device is recording
that way i can see what i did wrong and what i need to practise on.

I want to build something like Video Delay Instant Replay.
- https://play.google.com/store/apps/details?id=borama.co.mirrorcoach&hl=sv
- https://apps.apple.com/us/app/video-delay-instant-replay/id1135528824

...but with WebRTC and multiple devices to capture the video from another angle.
the replay device would be somewhere near and accessible and the other device would be somewhere else.

I don't not want to save the video recording on any device, and replay it with Media Source Extensions API. i have already tried that but it didn't work out well. 

My usecase is posted/discussed throughout  here:
- https://github.com/w3c/webrtc-pc/issues/2213
- https://gist.github.com/jimmywarting/2296342bfb7a31ad26068659d1fd9fc9#gistcomment-2945243 
- https://stackoverflow.com/questions/56510151/change-playout-delay-in-webrtc-stream

**Pull request merged**